gym/tf_od_api/workspace/train_core/core.sh

```diff
@@ -0,0 +1,9 @@
+cd /opt/tf_od_api/workspace/train_core
+
+echo "Evaluating the model"
+
+python model_main_tf2.py \
+--model_dir=/home/models/ssd_mobilenet_v2_fpnlite \
```

Comment:
I would say we might end up having our models in a folder other than `/home/...`. So, I'd put this as a variable at the beginning of the scrip to change if this folder changes in the future.

Reply:
the pipeline has been changed ...
about the /home, it refer to /home inside the docker container, I can not explain it here ...
but the idea is: we've mount and bind our workspace directory to /home of our container, doing so /home is like a pointer to our workspace